astroquery/esa/hubble/core.py

```diff
@@ -233,18 +205,17 @@ def cone_search(self, coordinates, radius=0.0, filename=None,
                    "ORDER BY PROPOSAL.PROPOSAL_ID "
                    "DESC".format(str(ra), str(dec), str(radiusInGrades)),
                    "RETURN_TYPE": str(output_format)}
-        result = self._handler.request('GET',
-                                       self.metadata_url,
-                                       params=payload,
-                                       cache=cache,
-                                       timeout=self.TIMEOUT)
+        response = self._request('GET',
+                                 self.metadata_url,
+                                 params=payload)
```

Comment:
were the `cache=` and `timeout=` keywords dropped here intentionally?  I suspect at least `TIMEOUT` was not